src/development/local.md

```diff
@@ -95,15 +90,16 @@ Close tunnels with: platform tunnel:close
 ```
 
 > **note**
-> platform tunnel: requires the pcntl and posix PHP extensions. Run `php -m | grep -E 'posix|pcntl'` to check if they're there.
+> The `platform tunnel:` commands require the pcntl and posix PHP extensions. Run `php -m | grep -E 'posix|pcntl'` to check if they're there.
 
-Now you can connect to the remote database normally, as if it were
-local.
+Now you can connect to the remote database normally, as if it were local.
 
 ```
 $ mysql --host=127.0.0.1 --port=30001 --user='user' --password='' --database='main'
 ```
 
+The specific port that each service uses may vary project to project, but it will not change within a single project unless you change your services or relationship definition.  That makes it safe to add a local-configuration file for your application that connects to, in this case, `localhost:30001` for the SQL database and `localhost:30000` for Redis.
```

Comment:
The local port may change at any time, it just depends whether it's already occupied by something else.

To stay up to date you could use the result of `platform tunnel:info --encode` as PLATFORM_RELATIONSHIPS, in PHP maybe:

```php
if ($relationships_encoded = shell_exec('platform tunnel:info --encode')) {
  $relationships = json_decode(base64_decode($relationships_encoded, TRUE);
  // ...
}
```

or run the server with `export PLATFORM_RELATIONSHIPS="$(platform tunnel:info --encode)"`

or something similar